issuetracker/issuetracker.go

```diff
@@ -90,14 +90,14 @@ func BaseURLFor(issueTracker config.IssueTracker, origin string) (string, error)
 func HealthcheckURL(issueTracker config.IssueTracker, origin string) (string, error) {
 	switch issueTracker {
 	case config.IssueTrackerGithub:
-		scheme, owner, repo := parseGithubDetails(origin)
+		scheme, owner, repo := ParseGithubDetails(origin)
 		return fmt.Sprintf("%s//api.github.com/repos/%s/%s", scheme, owner, repo), nil
 	default:
 		return "", ErrUnsupportedHealthCheck
 	}
 }
 
-func parseGithubDetails(origin string) (scheme, owner, repo string) {
+func ParseGithubDetails(origin string) (scheme, owner, repo string) {
```

Comment:
Why is this function made public? It isn't used anywhere outside this package.

Reply:
Sorry that this was my mistake. I returned it to private function.